_opensearch/segment-replication/configuration.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Segment Replication Configuration
+nav_order: 12
+parent: Segment Replication Feature
+grand_parent: Opensearch
+---
+
+## Segment Replication Configuration
+
+To enable the segment replication type, reference the steps below.
+
+### Enabling the feature flag
+
+There are several methods to enable segment replication depending on install type. You will also need to set replication strategy to `SEGMENT` when creating the index.
+
+#### Enable on a node using a tarball install
+
+The flag is toggled using a new jvm parameter that is set in either `OPENSEARCH_JAVA_OPTS` or in config/jvm.options.
+
+1. Option 1. Update config/jvm.options with:
```

Comment:
Change the period after "1" to a colon. Change "with" to "by [verb]ing the following [noun]".